odoo/addons/base/models/ir_model.py

```diff
@@ -2278,7 +2282,7 @@ def toggle_noupdate(self, model, res_id):
         """ Toggle the noupdate flag on the external id of the record """
         record = self.env[model].browse(res_id)
         if record.check_access_rights('write'):
-            for xid in  self.search([('model', '=', model), ('res_id', '=', res_id)]):
+            for xid in self.search([('model', '=', model), ('res_id', '=', res_id)]):
```

Comment:
The github police may find you for that :open_mouth:

Reply:
:shushing_face: There is almost no linter error in this file, I think it is cool to remove one here, as it is just a few lines below